src/pages/proposals/new-index.tsx

```diff
@@ -0,0 +1,128 @@
+						<div className="justify-between w-full px-4 rounded-2xl">
+							<div className="flex justify-between">
+								<p className="font-semibold text-xl py-4 text-white">Rebus Proposals</p>
+								<div className="flex flex-wrap items-center justify-center">
+									<p className="font-semibold text-xl px-1 text-primary-50">
+										<button
+											type="button"
+											className="hover:border-b-2 text-primary-50"
+											onClick={() => handleChange(ProposalStatuses.all)}>
+											<span>{proposals.length}</span>
+											<span
+												className={`${
+													active === ProposalStatuses.all ? 'active' : ''
+												} font-thin text-sm pl-1 text-primary-50`}>
+												{variables[lang].all}
+											</span>
+										</button>
+									</p>
+									<p className="font-semibold text-xl px-1 text-transparent bg-clip-text bg-gradient-pass">
+										<button
+											type="button"
+											className="cursor-pointer hover:border-b-2 text-transparent bg-clip-text bg-gradient-pass"
+											onClick={() => handleChange(ProposalStatuses.passed)}>
+											<span>{getProposalStatusCount(ProposalStatuses.passed)}</span>
+											<span
+												className={`${
+													active === ProposalStatuses.passed ? 'active' : ''
+												} font-thin text-sm pl-1 text-transparent bg-clip-text bg-gradient-pass`}>
+												{variables[lang].active}
+											</span>
+										</button>
+									</p>
+									<p className="font-semibold text-xl px-1 text-enabledGold">
+										<button
+											type="button"
+											className="cursor-pointer hover:border-b-2 text-enabledGold"
+											onClick={() => handleChange(ProposalStatuses.pending)}>
+											<span>{getProposalStatusCount(ProposalStatuses.pending)}</span>
+											<span
+												className={`${
+													active === ProposalStatuses.pending ? 'active' : ''
+												} font-thin text-sm pl-1 text-enabledGold`}>
+												{variables[lang].pending}
+											</span>
+										</button>
+									</p>
+									<p className="font-semibold text-xl px-1 text-transparent bg-clip-text bg-gradient-rejected">
+										<button
+											type="button"
+											className="cursor-pointer hover:border-b-2 text-transparent bg-clip-text bg-gradient-rejected"
+											onClick={() => handleChange(ProposalStatuses.rejected)}>
+											<span>{getProposalStatusCount(ProposalStatuses.rejected)}</span>
+											<span
+												className={`${
+													active === ProposalStatuses.rejected ? 'active' : ''
+												} font-thin text-sm pl-1 text-transparent bg-clip-text bg-gradient-rejected`}>
+												{variables[lang].closed}
+											</span>
+										</button>
+									</p>
+								</div>
+							</div>
+						</div>
+					</div>
+					<div className="cards-content md:p-4">
+						<NewCards proposals={[...filteredProposals]} />
```

Comment:
There's no reason to spread this array here I think

Reply:
Theres and array when I just pass proposal in